RadarSDK/RadarLocationManager.m

```diff
@@ -387,14 +386,12 @@ - (void)updateTrackingFromMeta:(RadarMeta *_Nullable)meta {
     if (meta) {
         if ([meta trackingOptions]) {
             [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug
-                                               message:@"Listening to server tracking options"];
-            [RadarSettings setListenToServerTrackingOptions:YES];
+                                               message:[NSString stringWithFormat:@"Setting remote tracking options | trackingOptions = %@", meta.trackingOptions]];
             [RadarSettings setRemoteTrackingOptions:[meta trackingOptions]];
         } else {
-            [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug
-                                               message:@"Not listening to server tracking options; reverting to fallback values"];
-            [RadarSettings setListenToServerTrackingOptions:NO];
             [RadarSettings removeRemoteTrackingOptions];
+            [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug
+                                               message:[NSString stringWithFormat:@"Removed remote tracking options | trackingOptions = %@", Radar.getTrackingOptions]];
```

Comment:
Should `Radar.getTrackingOptions` be `[Radar getTrackingOptions]`? Does it matter?

Reply:
They're functionally identical in Objective-C. I prefer the property-style call, but that's because I like Swift. :)